mllib/src/main/scala/org/apache/spark/ml/regression/AFTSurvivalRegression.scala

```diff
@@ -90,6 +108,12 @@ private[regression] trait AFTSurvivalRegressionParams extends Params
       SchemaUtils.checkColumnType(schema, $(censorCol), DoubleType)
       SchemaUtils.checkColumnType(schema, $(labelCol), DoubleType)
     }
+    if (hasQuantileProbabilities && hasQuantilesCol) {
+      SchemaUtils.appendColumn(schema, $(quantilesCol), new VectorUDT)
+    } else if (hasQuantileProbabilities || hasQuantilesCol) {
+      logWarning("The output will not include quantiles column. " +
+        "If you want to include, please set both quantileProbabilities and quantilesCol.")
+    }
```

Comment:
I think the sematic is clear to me. On the other hand, can we consider these two as a combo, i.e. 
case class QuantileParams(quantileProbabilities: Array[Double], quantilesCol: String)
Then the user will have to set both items simultaneously.

Not sure it will work but try to brain storm. :)